projects/storefrontapp-e2e-cypress/cypress/helpers/cart.ts

```diff
@@ -69,51 +82,71 @@ function goToProductFromSearch(id: string, mobile: boolean) {
   }
 }
 
-export function addProductToCartViaAutoComplete(mobile: boolean) {
-  const product = products[0];
+function checkMiniCartCount(expectedCount) {
+  return cy.get('cx-mini-cart').within(() => {
+    cy.get('.count').should('contain', expectedCount);
+  });
+}
 
-  goToProductFromSearch(product.code, mobile);
+export function validateEmptyCart() {
```

Comment:
Ask questions instead of giving orders.

**Bad**
Rename it to `emptyCartValidation`.

**Good**
What do you think about renaming this function to `emptyCartValidation`?

Reply:
If you recommend something and it might not be obvious why it's better for the other programmer also explain it better (or provide links to explanation).

Eg. Could we avoid this type assertion? Here is an explanation why it's not a good practice: https://timdeschryver.dev/blog/stop-misusing-typescript-type-assertions